### PR TITLE
fix(ObjectViewer): align styles + tests update

### PR DESCRIPTION
**Pull request by jmfrancois**
**What is the problem this PR is trying to solve?**

* openFirst do it's job but goes in conflict with state once you are playing with the widget
* badge is not aligned
* text in background is not aligned
* tests use storyshots

**What is the chosen solution to this problem?**

* align stuff
* remove openfirst
* tests are based on shallow rendering

screen shot 2017-10-06 at 15 45 08

**Please check if the PR fulfills these requirements**
- [x] The PR commit message follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [ ] Related design / discussions / pages (not in jira), if any, are all linked or available in the PR

**[ ] This PR introduces a breaking change**

### CODE REVIEW

File: output/components.eslint.txt [hunk 1]
```diff
@@ -17,8 +17,9 @@ The react/require-extension rule is deprecated. Please use the import/extensions
   4:2  error  Unexpected require()                                                                    global-require
 
 /home/travis/build/Talend/ui/packages/components/src/ObjectViewer/JSONLike/JSONLike.component.js
-  230:14  error  Expected parentheses around arrow function argument having a body with curly braces  arrow-parens
-  420:15  error  Expected parentheses around arrow function argument having a body with curly braces  arrow-parens
+   24:73  error  'selectedJsonpath' is defined but never used                                         no-unused-vars
```
Comment (sdwvit) [inline output/components.eslint.txt]: Hey, please fix

File: packages/components/src/ObjectViewer/JSONLike/JSONLike.component.js [hunk 2]
```diff
@@ -255,7 +249,7 @@ export function ComplexItem({ data, name, opened, edited, jsonpath, info, onSele
 						label={getDataAbstract(data)}
 						tooltipPlacement="right"
 					>
-						<sup className={badgeClasses}>{decoratedLength}</sup>
+						<sup className="badge">{decoratedLength}</sup>
```
Comment (sdwvit) [inline packages/components/src/ObjectViewer/JSONLike/JSONLike.component.js]: shouldn't it be theme['badge']?

Comment (jmfrancois) [reply packages/components/src/ObjectViewer/JSONLike/JSONLike.component.js]: there is no 'badge' class in the theme file, this is a bootstrap class.

File: packages/components/src/ObjectViewer/JSONLike/JSONLike.component.js [hunk 3]
```diff
@@ -21,7 +21,7 @@ function stopAndSelect(event, { onSelect, jsonpath }) {
 	onSelect(event, jsonpath);
 }
 
-export function NativeValue({ data, edit, onSelect, onChange, jsonpath, selectedJsonpath }) {
+export function NativeValue({ data, edit, onSelect, onChange, jsonpath }) {
```
Comment (acateland) [inline packages/components/src/ObjectViewer/JSONLike/JSONLike.component.js]: amen !

**Pull request merged**